.eslintrc

```diff
@@ -0,0 +1,152 @@
+{
+  "parser": "babel-eslint",
+
+  "root": true,
+
+  "env": {
+    // I write for browser
+    "browser": true,
+    // in CommonJS
+    "node": true,
+    "mocha": true
+  },
+
+  "extends": "eslint:recommended",
+
+  "globals": {
+    // having some trouble getting eslint to recognize es6 features
+    "Promise": false,
+    "Set": false,
+    "Map": false
+  },
+
+  // plugins let use use custom rules below
+  "plugins": [
+    "babel",
+    "react"
+  ],
+
+  "ecmaFeatures": {
+      "jsx": true,
+      "es6": true,
+    },
+
+  "rules": {
+    // 0 = off
+    // 1 = warn
+    // 2 = error
+
+    // React specifc rules
+    "react/jsx-boolean-value": 0,
+    "react/jsx-closing-bracket-location": 1,
+    "react/jsx-curly-spacing": [2, "always"],
+    "react/jsx-indent-props": [1, 2],
+    "react/jsx-no-undef": 1,
+    "react/jsx-uses-react": 1,
+    "react/jsx-uses-vars": 1,
+    "react/jsx-wrap-multilines": 1,
+    "react/react-in-jsx-scope": 1,
+    "react/prefer-es6-class": 1,
+    // no binding functions in render for perf
+    "react/jsx-no-bind": 1,
+
+    // handle async/await functions correctly
+    // handle object spread
+    "babel/object-shorthand": 1,
+
+    // handle potential errors
+    "brace-style": [2, "1tbs", { "allowSingleLine": true }],
+    "comma-dangle": [2, "always-multiline"],
+    "consistent-return": 0,
+    "indent": [2, 2, {"SwitchCase": 1}],
+    "keyword-spacing": [2],
+    "max-len": [1, 100, 2, {"ignoreComments": true}],
+    "no-cond-assign": [2, "always"],
+    "no-console": 0,
+    "no-constant-condition": 2,
+    "no-control-regex": 2,
+    "no-debugger": 2,
+    "no-dupe-args": 2,
+    "no-dupe-keys": 2,
+    "no-duplicate-case": 2,
+    "no-empty-character-class": 2,
+    "no-empty": 2,
+    "no-ex-assign": 2,
+    "no-extra-boolean-cast": 2,
+    "no-extra-semi": 2,
+    "no-func-assign": 2,
+    "no-invalid-regexp": 2,
+    "no-irregular-whitespace": 2,
+    "no-negated-in-lhs": 2,
+    "no-obj-calls": 2,
+    "no-regex-spaces": 2,
+    "no-sparse-arrays": 2,
+    "no-unexpected-multiline": 2,
+    "no-unreachable": 2,
+    "no-underscore-dangle": 0,
+    "quotes": [2, "single"],
```

Comment:
🔕  Seems like we're all in favor of double quotes now. Let's do a separate patch for all repos like that soon?